docs/content/guides/dagster/code-references.mdx

```diff
@@ -58,6 +60,43 @@ A link to the asset's source in `with_source_code_references.py` will then be vi
   />
 </center>
 
+### dbt assets
+
+Dagster's dbt integration can automatically attach references to the SQL files backing your dbt assets. To enable this feature, set the `enable_code_references` parameter to `True` in the <PyObject module="dagster_dbt" object="DagsterDbtTranslatorSettings" /> passed to your <PyObject module="dagster_dbt" object="DagsterDbtTranslator" />:
+
+```python file=/guides/dagster/code_references/with_dbt_code_references.py
+from pathlib import Path
+
+from dagster_dbt import (
+    DagsterDbtTranslator,
+    DagsterDbtTranslatorSettings,
+    DbtCliResource,
+    dbt_assets,
+)
+
+from dagster import AssetExecutionContext, Definitions
+from dagster._core.definitions.metadata import with_source_code_references
```

Comment:
We shouldn't put any private imports in docs

Reply:
Yeah, this is just for demonstration purposes until we expose these apis publicly - will do another pass once we do so